packages/ai-jsx/src/core/debug.tsx

```diff
@@ -150,10 +150,16 @@ export function debug(value: unknown, expandJSXChildren: boolean = true): string
           return `[${values.join(', ')}]`;
       }
     } else if (typeof value === 'object') {
+      let stringified;
+      try {
+        stringified = JSON.stringify(value);
+      } catch {
+        stringified = '{/* ... */}';
```

Comment:
What's the scenario where this comes up?

Reply:
If the OpenAI client gets JSON-stringified (which happens if you set the env bar to otel-trace every render in a fixie deployment) it throws.